controllers/materials/views/materials-fields.njk

```diff
@@ -78,14 +78,14 @@
                     hasOther = true
                 ) }}
             {% endif %}
-            {% if error %}<p class="error t6 u-margin-top">{{ error.msg }}</p>{% endif %}
+            {% if error %}<p class="u-margin-top">{{ error.msg }}</p>{% endif %}
         </div>
     </fieldset>
 
     {% if allowOther %}
         <label for="{{ otherId }}" class="u-visually-hidden">{{ __('global.forms.other') }}</label>
         <input type="text" name="{{ fieldName }}Other" id="{{ otherId }}" placeholder="{{ __('global.forms.other') }}" />
     {% endif %}
-    {% if error %}<p class="error t6 u-margin-top">{{ error.msg }}</p>{% endif %}
```

Comment:
These classes no longer exist, there's a wider job to finally make this form use the core form macros but for now removing these classes.

Reply:
Should we re-instate `.error`, at least? Right now it looks a bit minimal, eg:

![image](https://user-images.githubusercontent.com/394376/49949845-91683f80-feee-11e8-8bc5-7ee324a07d9d.png)

Could make it red again?

Reply:
Done